stl/inc/algorithm

```diff
@@ -6977,6 +7129,82 @@ void nth_element(_ExPo&&, _RanIt _First, _RanIt _Nth, _RanIt _Last) noexcept /*
     // not parallelized at present, parallelism expected to be feasible in a future release
     _STD nth_element(_First, _Nth, _Last);
 }
+
+#ifdef __cpp_lib_concepts
+namespace ranges {
+    // VARIABLE ranges::nth_element
+    class _Nth_element_fn : private _Not_quite_object {

```

Comment:
It looks like this is a range-ification of the classic algorithm, so it will be affected by #856 and #857. Those don't look quick to resolve, so I'm ok with this for now, just wanted to mention it.

Reply:
Yes it is, and yes it will.